charts/rancher-k3s-upgrader/0.3.1/values.yaml

```diff
@@ -0,0 +1,12 @@
+global:
+  systemDefaultRegistry: ""
+
+systemUpgradeController:
+  image:
+    repository: rancher/system-upgrade-controller
+    tag: v0.8.0-rc.1
```

Comment:
v0.8.0 has been tagged now, any reason we still want to use the rc?

Reply:
No, I didn't now the new version had been tagged. Updated.